src/deploy_inst.h

```diff
@@ -49,18 +49,18 @@ class DeployInst : public cyclus::Institution {
   std::vector<std::string> prototypes;
 
   #pragma cyclus var { \
-    "doc": "Time step on which to deploy agents given in prototype list (same order).", \
-    "uilabel": "Deployment times", \
-  }
-  std::vector<int> build_times;
-
-  #pragma cyclus var { \
     "doc": "Number of each prototype given in prototype list that should be deployed (same order).", \
     "uilabel": "Number to deploy", \
   }
   std::vector<int> n_build;
 
   #pragma cyclus var { \
+    "doc": "Time step on which to deploy agents given in prototype list (same order).", \
+    "uilabel": "Deployment times", \
+  }
+  std::vector<int> build_times;
```

Comment:
Actually - neither of them are unique.  The unique key consists of all four of: prototypes, n_build, build_times, and lifetimes (if specified).  So it doesn't much matter - except I think that prototypes should go first and lifetime last (because it's optional).